rest-dashboard_integration_test.go

```diff
@@ -42,6 +42,10 @@ func Test_Dashboard_CRUD(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	if boardLinks, err = client.SearchWithParams(ctx, sdk.WithSearchStarred(false)); err != nil {
```

Comment:
We should probably this and the other function into a "table test". Or, please move this a bit further down in this function and copy/paste the code that is after this which checks if the data is correct? Also, I see that it misses a check on `len` - there might be problems if there are no results. Could we add a check that `len(boardLinks) != 0` together with this change?

Reply:
> We should probably convert this and the other function into a "table test". Or, please move this a bit further down in this function and copy/paste the code that is after this which checks if the data is correct?

I've written some unit tests for `SearchDashboards` and `SearchWithParams` instead: https://github.com/grafana-tools/sdk/pull/86/commits/159268894149c9f5d45c9e8ad56dd7196d50876f
I think `/api/search` calls should be sufficiently covered with that.

> Also, I see that it misses a check on len - there might be problems if there are no results

Good catch! added to same commit.